src/Packetery/Module/Checkout.php

```diff
@@ -1038,4 +1053,16 @@ private function getTaxClassWithMaxRate() {
 		return $resultTaxClass;
 	}
 
+	/**
+	 * Tells cart price with VAT and without Coupons.
+	 *
+	 * @return float
+	 */
+	private function getCartPriceIncludingVatAndCoupons():float {
+		$cartTotalPrice    = $this->getCartPrice() + $this->getCartTax();
+		$cartContentTotal  = $this->getCartContentPrice() + $this->getCartContentTax();
```

Comment:
Vrátím se k otázce z PES-919 - proč nepoužít $cartContentTotal vždy? Podle WC dokumentace je $cartTotalPrice "before discounts" - nemůže to dělat neplechu?